core/sr-io/without_std.rs

```diff
@@ -104,7 +111,20 @@ pub fn storage(key: &[u8]) -> Option<Vec<u8>> {
 		if length == u32::max_value() {
 			None
 		} else {
-			Some(Vec::from_raw_parts(ptr, length as usize, length as usize))
+			Some(slice::from_raw_parts(ptr, length as usize).to_vec())
+		}
+	}
+}
+
+/// Get `key` from child storage and return a `Vec`, empty if there's a problem.
+pub fn child_storage(storage_key: &[u8], key: &[u8]) -> Option<Vec<u8>> {
+	let mut length: u32 = 0;
+	unsafe {
+		let ptr = ext_get_allocated_child_storage(storage_key.as_ptr(), storage_key.len() as u32, key.as_ptr(), key.len() as u32, &mut length);
+		if length == u32::max_value() {
+			None
+		} else {
+			Some(slice::from_raw_parts(ptr, length as usize).to_vec())
```

Comment:
One last time ^^
@pepyakin can't we call `ext_free` here for freeing `ptr`? 
If that not works, I would at least propose to add `TODO` over these lines that we do not forget these memory leaks.

Reply:
Yes, this is the only possible way, as long as `ext_*` make allocations with it.